Windows/Debugger/Debugger_MemoryDlg.h

```diff
@@ -6,19 +6,29 @@
 #include "Core/MemMap.h"
 
 #include "Core/Debugger/DebugInterface.h"
-
+#include "CtrlMemView.h"
 #include "Common/CommonWindows.h"
 
 class CMemoryDlg : public Dialog
 {
 private:
 	DebugInterface *cpu;
 	static RECT slRect;
-
+	RECT winRect, srRect;
+	CtrlMemView *memView;
+	HWND memViewHdl , symListHdl, editWnd, searchBoxHdl, srcListHdl;
 	BOOL DlgProc(UINT message, WPARAM wParam, LPARAM lParam);
+	
+	struct SearchResult {
```

Comment:
Is this struct used? A quick search didn't find any uses.

Reply:
I was thinking of extending the functionality a bit later, but doesn't have to be there now if you prefer